doc/sources/installation/installation-windows.rst

```diff
@@ -140,146 +153,150 @@ For installation instructions, see :ref:`install-win-dist`. See also :ref:`dev-i
 
 .. _kivy-dependencies:
 
-Kivy's dependencies
--------------------
+Installing Kivy's dependencies
+--------------------------
 
-We offer wheels for Kivy and its dependencies, separately, so only desired
-dependencies need be installed. The dependencies are offered as
-optional sub-packages of kivy_deps, e.g. ``kivy_deps.sdl2``.
+We offer the above wheels for the Kivy core and for its below dependencies separately, so that you can install only the desired
+dependencies. The dependencies are offered as sub-packages, starting with ``kivy_deps``, for example ``kivy_deps.sdl2``.
 
 .. note::
 
-    In Kivy 1.11.0 we transitioned the kivy Windows dependencies from the
-    `kivy.deps.xxx` namespace stored under `kivy/deps/xxx` to the
-    `kivy_deps.xxx` namespace stored under `kivy_deps/xxx`.
+    In Kivy 1.11.0 we replaced the dot in ``kivy.deps`` with an underscore. So, instead of ``kivy.deps.xxx``, stored under ``kivy/deps/xxx`` it is now ``kivy_deps.xxx``, stored under ``kivy_deps/xxx``.
     See `here <https://github.com/kivy/kivy/wiki/Moving-kivy.garden.xxx-to-kivy_garden.xxx-and-kivy.deps.xxx-to-kivy_deps.xxx#kivy-deps>`_
     for more details.
 
-On Windows, we provide the following dependency wheels:
+So, the following are the dependency wheels which we provide for Windows:
+
+* `gstreamer <https://gstreamer.freedesktop.org>`_ (optional)
+
+  `gstreamer` is an optional dependency which is only needed for audio/video support.
+  It can be installed with  ``python -m pip install kivy_deps.sdl2``
 
-* `gstreamer <https://gstreamer.freedesktop.org>`_ for audio and video.
+* `ffpyplayer <https://pypi.org/project/ffpyplayer/>`_ (optional)
+
+  `ffpyplayer` is an alternative optional dependency for audio or video.
+  It can be installed with ``python -m pip install ffpyplayer``.
 
-  ``gstreamer`` is an optional dependency which is only needed for audio/video support.
-  ``ffpyplayer`` is an alternate dependency for audio or video. It can be installed with
-  ``pip install ffpyplayer``.
 * `glew <http://glew.sourceforge.net/>`_ and/or
-  `angle <https://github.com/Microsoft/angle>`_ for OpenGL.
+  `angle <https://github.com/Microsoft/angle>`_
+  
+  These are for `OpenGL <https://en.wikipedia.org/wiki/OpenGL>`_. They can be installed with ``python -m pip install kivy_deps.glew`` and/or ``python -m pip install kivy_deps.angle``.
```

Comment:
You have added the `python -m pip install kivy_deps.xxx`  instructions here. In the above examples, when kivy was installed using `kivy[base]`, that automatically installs them, so I'm not sure we need to list them here.

Perhaps this table is helpful and should be included. It indicates what is installed with the given keyword as listed here https://github.com/kivy/kivy/blob/master/setup.cfg :

|                     | none | base | full |
|---------------------|------|------|------|
| kivy_deps.sdl2      | n    | y    | y    |
| kivy_deps.glew      | n    | y    | y    |
| kivy_deps.angle     | n    | y    | y    |
| kivy_deps.gstreamer | n    | n    | y    |

Reply:
Good idea. Added that table. Now my fingers hurt.